tests/middleware_exceptions/tests.py

```diff
@@ -162,11 +162,23 @@ def assert_exceptions_handled(self, url, errors, extra_error=None):
             self.assertEqual(value.args, (error, ))
 
     def assert_middleware_usage(self, middleware, request, view, template_response, response, exception):
-        self.assertEqual(middleware.process_request_called, request)
-        self.assertEqual(middleware.process_view_called, view)
-        self.assertEqual(middleware.process_template_response_called, template_response)
-        self.assertEqual(middleware.process_response_called, response)
-        self.assertEqual(middleware.process_exception_called, exception)
+        self.assertEqual(
+            (
+                middleware.__class__.__name__,
```

Comment:
Is this change to aid debugging (get the middleware's class name in the message)? If so, maybe add a comment about that and make the change in a separate commit.